packages/mds-agency-cache/index.ts

```diff
@@ -45,64 +45,19 @@ const { env } = process
 
 const { unflatten } = flatten
 
-declare module 'redis' {
-  interface RedisClient {
-    dbsizeAsync: () => Promise<number>
-    delAsync: (...arg1: string[]) => Promise<number>
-    flushdbAsync: () => Promise<'OK'>
-    hdelAsync: (...args: (string | number)[]) => Promise<number>
-    hgetallAsync: (arg1: string) => Promise<{ [key: string]: string }>
-    hgetAsync: (key: string, field: string) => Promise<string>
-    hscanAsync: (key: string, cursor: number, condition: string, pattern: string) => Promise<[string, string[]]>
-    hsetAsync: (key: string, field: string, value: string) => Promise<number>
-    hmsetAsync: (...args: unknown[]) => Promise<'OK'>
-    infoAsync: () => Promise<string>
-    keysAsync: (arg1: string) => Promise<string[]>
-    zaddAsync: (arg1: string | number, arg2: number, arg3: string) => Promise<number>
-    zrangebyscoreAsync: (key: string, min: number | string, max: number | string) => Promise<string[]>
-    georadiusAsync: (key: string, longitude: number, latitude: number, radius: number, unit: string) => Promise<UUID[]>
-  }
-  interface Multi {
-    hgetallAsync: (arg1: string) => Promise<{ [key: string]: string }>
-    execAsync: () => Promise<object[]>
-  }
-}
-
-bluebird.promisifyAll(redis.RedisClient.prototype)
-bluebird.promisifyAll(redis.Multi.prototype)
-
-let cachedClient: redis.RedisClient | null
+const client = RedisCache()
```

Comment:
I think this should be something more like `RedisCache().initialize()`.

Reply:
See changes